tests/integration/test_outcome.py

```diff
@@ -1648,19 +1648,21 @@ def test_global_rate_limit(
     projectconfig = mini_sentry.add_full_project_config(project_id)
     mini_sentry.add_dsn_key_to_project(project_id)
 
+    ts = datetime.utcnow().timestamp()
+    # Ensures we begin at the start of a slot so we don't go to next slot in the middle of the test
+    window = ts
```

Comment:
hmm,  i mainly did it to make documentation easier, since it's a bit awkward to put a long doc sentence in the bucket json, but yeah I removed that variable now